src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { sumValues, sumVault } from './lib/addingFunctions/addUp'
+import { getVertoState } from './lib/warp/getVertoState'
+import * as fs from 'fs'
+
+
+// tate small numbers
+const firstNumbers = JSON.parse(fs.readFileSync('/Users/Lorimer/Desktop/Othent.io/VertoVaultBalances/src/lib/tateStuff/firstNumbers.json', 'utf-8'))
+const firstNumbersAdded = sumValues(firstNumbers);
+console.log('firstNumbersAdded', firstNumbersAdded);
+
+
+// tate state
+const tateState = JSON.parse(fs.readFileSync('/Users/Lorimer/Desktop/Othent.io/VertoVaultBalances/src/lib/tateStuff/tateFile.json', 'utf-8'))
+const tateStateAdded = await sumVault(tateState.vault)
+console.log('tateStateAdded', tateStateAdded)
+
+
+// smartweave state
+const smartweaveState = JSON.parse(fs.readFileSync('/Users/Lorimer/Desktop/Othent.io/VertoVaultBalances/smartweaveState.json', 'utf-8'))
+const smartweaveStateAdded = await sumVault(smartweaveState.vault)
+console.log('smartweaveStateAdded', smartweaveStateAdded)
+
+
+// tom tool state
+const tomToolState = JSON.parse(fs.readFileSync('/Users/Lorimer/Desktop/Othent.io/VertoVaultBalances/src/lib/tomTool/tomToolRes.json', 'utf-8'))
+const tomToolStateAdded = await sumVault(tomToolState.vault)
+console.log('tomToolStateAdded', tomToolStateAdded)
+
+
+// PPE state
+const ppeState = JSON.parse(fs.readFileSync('/Users/Lorimer/Desktop/Othent.io/VertoVaultBalances/src/lib/ppeRecommendation/ppeRecommendation.json', 'utf-8'))
+const ppeStateAdded = await sumVault(ppeState.state.vault)
+console.log('ppeStateAdded', ppeStateAdded)
```

Comment:
I'll have to create a "Lorimer" account on a mac to test it with those jsons 😛 🤣

Reply:
Lol me being lazy soz